Add oreganized.json

Freshah opened the pull request:
Hope I did this correctly!  simple addition to the defaults for Lead/Silver compat. 

![2022-04-01_20 13 40](https://user-images.githubusercontent.com/71207812/161670260-02fd1eb8-f890-487b-8f08-278f0061c7ff.png)

Freshah committed:
Add files via upload

Oreganized compat

lukebemish commented on common/src/main/resources/default_configs/oreganized.json:
```diff
@@ -0,0 +1,39 @@
+      "id": "lead_ore",
+      "stone": [
+        "stone",
+        "deepslate"
```
Every provided stone should have a corresponding block_id.

lukebemish commented on common/src/main/resources/default_configs/oreganized.json:
```diff
@@ -0,0 +1,39 @@
+      "id": "silver_ore",
+      "stone": [
+        "stone",
+        "deepslate"
```
Same as above.

lukebemish reviewed (changes requested):
Looks good; could you either provide a block ID for the deepstone variants, if they already exist, or if they don't remove it from the stone list?

lukebemish commented:
If you don't mind me asking, what are the bottom left two stones in the image?

Freshah commented:
I'll fix that right up!  The stones are Shale and Jasper from Quark, yeah.

lukebemish commented:
Cool! I'm glad the texture extraction works as well here as it does.

Freshah committed:
Update oreganized.json

removed deepslate refs.  leftovers from copying default immersiveengineering.json

lukebemish commented:
Thank you!

Freshah commented:
The variants exist but don't spawn so after release I may have to update.  If I end up using more stone and ore mods I'll keep contributing.

lukebemish commented:
They should spawn, though it probably depends how Oreganized is doing its ore gen. After it's released on 1.18.2, feel free to file a big report if they're not spawning.
